cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/namespace/DefaultNamespaceAdmin.java

```diff
@@ -194,7 +195,14 @@ public synchronized void create(final NamespaceMeta metadata) throws Exception {
 
       // if needed, run master environment specific logic
       MasterEnvironment masterEnv = MasterEnvironments.getMasterEnvironment();
-      if (masterEnv != null) {
+      Map<String, String> properties = cConf.getPropsWithPrefix(Constants.Namespace
```

Comment:
let's just have the single property rather than have the ability to shut it on/off per master environment. Makes the code easier to read, and anything environment specific should be done within the MasterEnvironment implementation and not at this layer anyway.

Reply:
Done